British-Oceanographic-Data-Centre/COAsT-site
# Example edits

Restructured Examples section.
Also added Overview text
Getting started update

To test:
```
git checkout example_edits
hugo server
```

Examples has sections (pages):

* The NEMO class
An introduction to the NEMO class. Loading variables and grid information.

* Transect analysis
An introduction to transect analysis (doesn't exist as in a state of flux)

* Stratification diagnostics
An introduction to stratification diagnostics

* Code snippets
This page is out of date…. To be deleted.

* Unit testing
Further information to describe COAsT’s functionality.

* Configuration Gallery
Example scripts and gallery for other NEMO configurations. Scripts from example_scripts


Also I updated "Getting Started" snippet from ``conda install -c bodc COAsT`` to ``conda install -c conda-forge -c bodc coast`` in line with Matt's instruction on Campfire

This is probably sufficient documentation to get us launched.


AC:
- [ ] It looks OK running ``hugo server``
- [ ] No blinding errors
- [ ] Matt: Correct ``conda install`` edit?

More work...
I can see that there is more to be done, but I don't think it is urgent. I've put some of these on a new ticket https://github.com/British-Oceanographic-Data-Centre/COAsT/issues/165  that can be split into ticket items later.
